src/components/typography/typography.d.ts

```diff
@@ -1,34 +1,45 @@
-import * as React from 'react';
-import { SpacingProps, ColorProps } from '../../utils/helpers/options-helper';
+import { StyledComponentProps } from "styled-components";
+import { SpacingProps, ColorProps } from "../../utils/helpers/options-helper";
 
 export interface TypographyProps extends SpacingProps, ColorProps {
+  /** Override the variant component */
   as?: string;
-  variant?: 'h1-large'|
-  'h1'|
-  'h2'|
-  'h3'|
-  'h4'|
-  'h5'|
-  'segment-header'|
-  'segment-header-small'|
-  'segment-subheader'|
-  'segment-subheader-alt'|
-  'p'|
-  'small'|
-  'big'|
-  'sup'|
-  'sub'|
-  'strong'|
-  'b'|
-  'em';
+  /** The visual style to apply to the component */
+  variant?:
+    | "h1-large"
+    | "h1"
+    | "h2"
+    | "h3"
+    | "h4"
+    | "h5"
+    | "segment-header"
+    | "segment-header-small"
+    | "segment-subheader"
+    | "segment-subheader-alt"
+    | "p"
+    | "small"
+    | "big"
+    | "sup"
+    | "sub"
+    | "strong"
+    | "b"
+    | "em";
+  /** Override the variant font-size */
   fontsize?: string;
```

Comment:
@grabkowski 
Noticed that this should be `fontSize`, not `fontsize`. I can raise an issue if you'd prefer but thought it would be quicker to raise it here

Reply:
good catch, I have missed this one, I'll just push that in this PR